### PR TITLE
[#2167] Add production, default configuration via .env

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
@thisandagain R?

**Comment by k88hudson** [comment]
Actually @gvn could you take a look?

File: README.md [hunk 1]
```diff
@@ -51,6 +51,29 @@ Forum: [https://groups.google.com/forum/#!forum/mozilla.webmaker](https://groups
 
 ---
 
+## Configuration
+
+### Changing configuration
+
+You can see all the default configuration in `config/defaults.env`. In order to change something, create an env file in your root directory and format configuration as follows:
```
Comment (gvn) [inline README.md]: Just for clarity, might want to mention that you have to name it `.env`

Comment (k88hudson) [inline README.md]: good call

**Comment by gvn** [comment]
Seems good. Build still works for me. Just one suggestion!

**Pull request merged**